CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-# March 2021, the oldest we have to support : Ununtu 18.04 LTS
-cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+# Dec 2022 Bumped min (ubuntu 20.04: 3.16.3-1ubuntu1.20.04.1)
```

Comment:
It would be better to add the link to the discussion to the comment. This way, people who might have issues will quickly find the discussion and why the decision was made to raise the version requirement.

Reply:
But the comment already states unambiguously the reason .. ?

Reply:
Because in the past we did it not optimal does not mean we have to continue.
Be aware that there are non Linux users. For example I have never seen Ubuntu in real life. We cannot expect people to understand that we follow a certain Linux distribution. So just input the link to the forum and everybody will find the discussion, can participate and understands the decision.